updates/testing.json

```diff
@@ -0,0 +1,19 @@
+{
+  "updates": {
+    "barriers": [],
+    "deadends": [
+      {
+        "version": "30.20190716.1",
+        "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/215"
+      }
+    ],
+    "rollouts": [
+      {
+        "version": "30.20190801.0",
+        "start_epoch": "1565017200",
+        "start_value": "0.0",
```

Comment:
Super minor: `value` seems a bit generic. Should this be e.g. `start_percentage` instead?

Reply:
Yup.Though it goes from `0.0` to `1.0`. Is "percentage" ok or is there a better term? Also, this is not a percentage over the global nodes population, but just a "rollout progression value".

Reply:
I think `percentage` would be fine in the context of `0.0` to `1.0` but I'm not super attached to either way personally.

Reply:
Out-of-band discussion: we'll go with `start_percentage`.